src/inference/src/cpp/core.cpp

```diff
@@ -83,6 +83,7 @@ std::map<std::string, Version> Core::get_versions(const std::string& device_name
 std::shared_ptr<ov::Model> Core::read_model(const std::wstring& model_path,
                                             const std::wstring& bin_path,
                                             const ov::AnyMap& properties) const {
+    OV_ITT_SCOPED_REGION_BASE(ov::itt::domains::OV, "Read model");
```

Comment:
Remove them, they are in core implementation

Reply:
They were one of the key requirements for this feature, actually: "Read model", "Compile model" and "Execution model" are the steps that are intended to be visible on "BASE" tracing level